src/commands/networks/removeNetwork.ts

```diff
@@ -13,11 +14,14 @@ export async function removeNetwork(context: IActionContext, node?: NetworkTreeI
     nodes = await multiSelectNodes(
         { ...context, suppressCreatePick: true, noItemFoundErrorMessage: 'No networks are available to remove' },
         ext.networksTree,
-        new RegExp(NetworkTreeItem.contextValue, 'i'),
+        NetworkTreeItem.customNetworkRegExp,
         node,
         nodes
     );
-
+    if (nodes.some(n => builtInNetworks.includes(n.networkName))) {
+        await ext.ui.showWarningMessage("The built-in networks 'bridge', 'host', and 'none' cannot be removed.");
```

Comment:
Nitpick: by convention at least in this repo, we try to use single-quotes on strings. `'The built-in networks \'bridge\', \'host\', and \'none\' cannot be removed.'`